core/utils/poly-look/src/css/themes.css

```diff
@@ -19,3 +19,10 @@
   --theme-primary-color: var(--color-indigo-950);
   --theme-secondary-color: var(--color-gray-50);
 }
+.poly-theme h1,
+.poly-theme h2,
+.poly-theme h3,
+.poly-theme h4,
+.poly-theme h5 {
+  font-weight: var(--font-weight-regular);
+}
```

Comment:
@RichardSto did you have this in mind  or should I change some namings here?

Reply:
Yeah thats good! Also want to add the font-family there?

Reply:
we have two possible families (Jost Bold and Jost Medium) . Should I define the most common for h-tag's (Medium) here and the Jost Bold directly where it is used?

Reply:
I would suggest use h1,h2 with bold and the others with medium?

Reply:
I could do that, although we have some h1 jost Medium in the facebookImporter (which have "local" Jost Medium definitions anyways), but I am not sure if that really fits to our actuak usage of h-tags. My impression is that jost bold is more the exception then a regularly used bold-option

Reply:
I see what you mean. Yet I think from a logical perspective h1 and 2 should have the boldest settings and we use h1 and h2 quite infrequent anyway. But I'd be fine not doing this in this PR

Reply:
OKay, good. I'm fine with that, we might have to keep that in mind. But if we are working on a good structure and you think this it, then I am happy to implement!

Reply:
Argh. It's just too much stuff breaking. I will merge this PR and set up another one, where I focus on reimplementing font-families. Or maybe we should have a little discussion with the front-enders beforehand